src/app/login/login.hbs

```diff
@@ -0,0 +1,17 @@
+<div class="login-container">
+    <div class="login-text-box">
+        <h3>Login</h3>
+        <p style="font-size: 80%;">Get access to your Orders, Wishlist and Recommendations</p>
+    </div>
+    <form class="login-form">
```

Comment:
How will you will ensure security of user data? This form doesn't have even basic best practices backed in.